examples/IMU/imu_rotation_vector.py

```diff
@@ -5,6 +5,16 @@
 import time
 import math
 
+device = dai.Device()
+
+imuVersion = device.getConnectedIMUVersion()
+imuFirmwareVersion = device.getIMUFirmwareVersion()
+print(f"IMU version: {imuVersion}, firmware version: {imuFirmwareVersion}")
+
+if imuVersion != dai.IMUVersion.BNO086:
```

Comment:
This feels weird. I would prefer dai.IMUModel or dai.IMUChip or something not "version".

A version is 1.2.3 and uses daiVersion struct. This API call and result enumeration is not a version to me